xsd/siri_model/siri_estimatedVehicleJourney.xsd

```diff
@@ -363,6 +363,11 @@ the original journey and the nature of the difference.</xsd:documentation>
 					<xsd:documentation>Relations of the journey with other journeys, e.g., in case a joining/splitting takes place or the journey substitutes for another one etc.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="UpdatedServiceLinks" type="ServiceLinkViewsStructure" minOccurs="0">
```

Comment:
This is not acceptable for me. This will cause implementations to provide UpdatedServiceLinks for every EstimatedTimetable Update.